models/ConfigureForm.php

```diff
@@ -14,9 +14,12 @@
 
 class ConfigureForm extends Model
 {
+    const REGEX_AGE = '/^(1[89]|[2-9]\d)$';
```

Comment:
I'm not really that good with REGEX so it would be nice if I could get some help on this :thinking:

Reply:
Why even use a regex here? Just use a number validation with min/max: https://www.yiiframework.com/doc/guide/2.0/en/tutorial-core-validators#number or do you just want to allow specific values here? But if you want to learn regex you can test your expressions for example at https://regex101.com/

Reply:
I wanted to use a regex age validation but guess I failed here even in my test run same result 🤔

Reply:
Removed